src/auth.ts

```diff
@@ -115,6 +115,7 @@ export async function authorize(useLocalhost: boolean, writeToOwnKey: boolean) {
     const token = await (useLocalhost ? authorizeWithLocalhost() : authorizeWithoutLocalhost());
     await (writeToOwnKey ? DOTFILE.RC_LOCAL.write(token) : DOTFILE.RC.write(token));
     console.log(LOG.AUTH_SUCCESSFUL);
+    process.exit(0);
```

Comment:
A comment as to why this is needed wouldn't hurt.

Reply:
👍